django_netjsonconfig/static/django-netjsonconfig/js/widget.js

```diff
@@ -58,10 +59,11 @@
         // add controls to the editor header
         advanced.find('.jsoneditor-menu')
             .append($('<a href="javascript:;" class="jsoneditor-poweredBy"><img class="icon" src="/static/admin/img/icon-deletelink.svg" /> back to normal mode</a>').click(function(){
-                toggleFullScreen();
+                if(isValidJson(editor))
```

Comment:
use curly brackets around the if block in order to avoid preventing the use of js minifiers.